### ISSUE TITLE
Page for: Advantech Adam 6717

### ISSUE THREAD

**Issue by ZJvandeWeg** [FlowFuse/website#2579]
### Description

When FlowFuse tests hardware for customers, we've opened up our research finding before. In the case for the Adam 6717 the research could be found at: https://docs.google.com/document/d/1fNvD56qJnepWqXomufpdDYfUKRX_7EFLqX-Act11pt8/edit

Most of which could be cleaned up a bit and put on a page much like: https://flowfuse.com/node-red/hardware/siemens-iot-2050/

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by Steve-Mcl** [comment]
ZJ, Joe has just pointed this out to me.

My apologies for missing this and not feeding back here but I did feedback to Zack at the time that the device is not _really_ suitable as a device-agent device for FlowFuse

TL;DR

* The device CAN run Device Agent
* It has a severe lack of RAM
* CPU is very slow with updates taking between three and five minutes
* It has a severe lack of storage space ( without adding an SD card it is pretty much a no go) ( and simply adding an SD card as not super feasible because you need a good quality industrial strength SD card to avoid corruption and data loss)
*  It is our opinion that there will be no update to node 18 due to the Architecture (32 bit) and kernel version (4.x) used on the device
    * This prevents the device from running anything greater than node red 3.x
    * This situation will only get worse with time
* Significant work will be required from our end getting this up and running in a stable production ready manner. We may have to build our own image or generate scripts that remove the existing node and node red versions and replace them with our own.

**Comment by ZJvandeWeg** [comment]
@sumitshinde-84 Can you collaborate with @Steve-Mcl to get a hardware page of the ground like with the others? I think the hardware page itself should show a checkmark for other hardware the support FlowFuse's agent comfortably, and a red cross otherwise. Please put that on the other pages too.
